bio_hansel/main.py

```diff
@@ -246,8 +249,8 @@ def main():
 
     if output_tile_results:
         if len(dfs) > 0:
-            dfall = pd.concat(dfs)  # type: pd.DataFrame
-            dfall = dfall.sort_values(by='is_pos_tile', ascending=False)
+            dfall = pd.concat([df.sort_values('is_pos_tile', ascending=False) for df in dfs], sort=False)  # type: pd.DataFrame
```

Comment:
Just something else I noticed, since we're explicitly targeting Python 3.6 or greater (we say so in the README and the [Conda recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/bio_hansel/meta.yaml#L16) targets Python 3.6). I think we can use a nicer syntax for optional typing in Python:

```python
dfall: pd.DataFrame = pd.concat(...)
# or 
dfs: List[pd.DataFrame] = [...]
# or
some_value: int = get_some_value()
# etc
```

See [Python typing module docs](https://docs.python.org/3.6/library/typing.html)  for more details.

It might be a good change to clean up the code a bit.

Reply:
I just adjusted the two mentioned but more of them can probably be adjusted looking more closely at it. I can keep doing it if wanted/create an issue for it and look at all of the files

Reply:
I think this is good for now. The nicer optional typing syntax can be added in later PRs. 

Thanks for making the changes! This PR looks good and will approve changes for merge.